src/renderer/widgets/AddProxyModal/ui/AddProxy.tsx

```diff
@@ -29,6 +34,19 @@ export const AddProxy = () => {
 
   if (addProxyUtils.isSubmitStep(step)) return <OperationSubmit isOpen={isModalOpen} onClose={closeModal} />;
 
+  if (addProxyUtils.isBasketStep(step)) {
+    setTimeout(() => closeBasketModal(), 1450);
```

Comment:
Why do we need to close basketModals on setTimeout? I'm not in the basket context, so could you please explain

Reply:
Requirement from designers